packages/extension/src/utils/tests/getTab.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies.
+ */
+import { getTab } from '../getTab';
+const dummyTab = {
```

Comment:
@amovar18 Can you checkout [this](https://github.com/acvetkov/sinon-chrome) package i think we are already using it and it provides mocking for extension related APIs

Reply:
@ayushnirwal  It does provide mocking capability for tests but the point for this test is to check given the `tabID` do we get the details correctly or not.